geode-assembly/src/test/java/org/apache/geode/rest/internal/web/RestSecurityIntegrationTest.java

```diff
@@ -94,7 +94,7 @@ public static void before() throws Exception {
   @Test
   public void testFunctions() throws Exception {
     String json = "{\"@type\":\"double\",\"@value\":210}";
-
+    Thread.sleep(500000);
```

Comment:
Yeah, I think we should be very adamant about not adding sleeps. Please change this to an Awaitility call that's awaiting whatever condition this sleep is trying to let happen.